Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/fragments/EmulationFragment.java

```diff
@@ -206,6 +208,10 @@ public void surfaceCreated(@NonNull SurfaceHolder holder)
   {
     // We purposely don't do anything here.
     // All work is done in surfaceChanged, which we are guaranteed to get even for surface creation.
```

Comment:
This comment is no longer accurate. Could you clarify it, replacing "anything" with mentioning the part we're still not doing here?